### PR TITLE
PJR-179/pjr-starter

### PR DESCRIPTION
**Pull request by kristojorg**
This PR copies over the OEH app into a PJR directory and then modifies it to try and remove everything OE specific, including the UI for now.

### CODE REVIEW

**Comment by kristojorg** [comment]
Sooo unfortunately I'm somehow unable to run cypress here (or in the web reader). I get a rosetta error:
```
rosetta error: Attachment of code signature supplement failed: 1 
 /var/db/oah/af5cd6b197f2001f6d58ecc6710bda014907088a0418bac03088f7a9b21df931/5f5f930fd4a3b44a5d9821f76e84e8a00da8b14febbdaa37a0a5237981505c75/Cypress.aot
The Test Runner unexpectedly exited via a exit event with signal SIGABRT
```

Rosetta is what Mac OS uses to translate intel apps to apple silicon, so I imagine it has to do with that. According to Cypress, version 10.x and up is natively compatible with apple silicon. It might be worth trying to upgrade? In their [changelog](https://docs.cypress.io/guides/references/changelog#10-0-0) they say you can run `cypress open` and then click update and it will update a bunch of stuff automatically for you. I'd love to do that but am unable to open cypress. Would someone be willing to branch off this branch, run `cypress open`, follow the update prompts and then commit the result? I can follow that up by fixing anything else manually once I can run it on my machine.

Let me know @Toxiapo @samanthaandrews thank you!

Also it looks like cypress has moved on to version 12 now (we are on v9). Do y'all think it's worth trying to upgrade all the way?

**Comment by samanthaandrews** [comment]
Yup I can work on this. It's weird bc I also have an M1 mac but can still open and run Cypress on this branch. But we should probably upgrade anyway.

**Comment by kristojorg** [comment]
I rebased this on top of my epub-to-webpub PR since that included some jest fixes that I needed to pull in. Hopefully we can get that one merged soon

**Comment by kristojorg** [comment]
This is ready for review. I don't think y'all need to review too specifically these initial PJR changes, mainly just make sure I'm not messing up anything on your side.

**Comment by samanthaandrews** [comment]
My plan is to pull down the branch and make sure I can get it to run in the web and on an iOS & Android emulator. Should all of these things work as of right now @kristojorg ?

**Comment by kristojorg** [comment]
Yes @samanthaandrews it should work at least as well as the OEH app. As we discussed, we are all experiencing issues running the apps on android with live reload. I have submitted a support ticket about this. Otherwise, I think this is ready to merge.

File: apps/oeh/android/capacitor.settings.gradle [hunk 1]
```diff
@@ -6,7 +6,7 @@ include ':capacitor-filesystem'
 project(':capacitor-filesystem').projectDir = new File('../../../node_modules/@capacitor/filesystem/android')
 
 include ':capacitor-preferences'
-project(':capacitor-preferences').projectDir = new File('../node_modules/@capacitor/preferences/android')
+project(':capacitor-preferences').projectDir = new File('../../../node_modules/@capacitor/preferences/android')
```
Comment (samanthaandrews) [inline apps/oeh/android/capacitor.settings.gradle:9]: this change in the `oeh` android app seems weird but maybe it's fine

Comment (samanthaandrews) [reply apps/oeh/android/capacitor.settings.gradle:9]: ah i think it's bc i updated the capacitor cli package

File: packages/epub-to-webpub/package.json [hunk 2]
```diff
@@ -15,12 +15,8 @@
     "lint": "eslint src/** --fix",
     "build": "tsup-node",
     "dev": "npm run start",
-    "start": "tsup-node --watch"
-  },
-  "bin": {
-    "epub-to-webpub": "./dist/cli/index.js",
-    "download-webpub": "./dist/cli/download-webpub.js",
-    "decrypt-webpub": "./dist/cli/decrypt-webpub.js"
```
Comment (samanthaandrews) [inline packages/epub-to-webpub/package.json:23]: just checking that these changes are intentional?

**Review by samanthaandrews — APPROVED** [review]
LGTM. Just want to make sure that anything that changed outside of the `apps/pjr` directory is correct.

**Pull request merged**